.github/workflows/release.yaml

```diff
@@ -23,3 +23,46 @@ jobs:
           args: release --clean
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  build-and-push:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Setup Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.ROLE_TO_ASSUME }}
+          role-session-name: "mackerel-sql-metric-collector:${{ github.run_id }}-${{ github.run_number }}"
+          aws-region: ap-northeast-1
+      - name: Login to Public ECR
+        uses: docker/login-action@v3
+        with:
+          registry: public.ecr.aws
+        env:
+          AWS_REGION: us-east-1
+
+      - name: Get version from tag
+        id: get-version
+        run: echo "VERSION=${GITHUB_REF/regs\/tags\//}" >> "$GITHUB_OUTPUT"
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v5
+        with:
```

Comment:
[should] `provenance: false` option would be required for ECR. cf.) https://zenn.dev/mokmok_dev/articles/0c7437929ece80

Reply:
I fixed it: https://github.com/mackerelio-labs/mackerel-sql-metric-collector/pull/6/commits/8340d00ac4fe6b6a7a6346f396dcfcf9199a9e52